new-hunt-setup.md

```diff
@@ -131,11 +131,6 @@ Global Google Drive, Sheets, and API settings for automatic sheets creation:
 * `GOOGLE_API_PRIVATE_KEY` - the private key for the key you added, with newlines replaced with `\n` (should be the value of the `private_key` field in the downloaded JSON when you [created the key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys); should look something like `-----BEGIN ... KEY-----\n...<long base64-encoded key>...\n-----END ... KEY-----\n`)
 * `GOOGLE_API_X509_CERT_URL` - the value of the `client_x509_cert_url` field in the downloaded JSON when you [created the key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys)
 
-Hunt-specific Google settings:
-
-* `GOOGLE_DRIVE_HUNT_FOLDER_ID` - the id of your Google Drive folder, should be part of the URL (`https://drive.google.com/drive/folders/<folder_id>`). This environment variable is used for getting the list of allowed emails for OAuth. Please keep consistent with the value in HuntSettings. Deprecating this variable is tracked in #662.
```

Comment:
are we able to delete the code in settings.py that read these environment variables? https://github.com/cardinalitypuzzles/cardboard/pull/753/files#diff-c003920b4e2c2658c5433b6599e1a2c0c21fd4abfeb45a7b32a8850b4a47f298L209

Reply:
oh we still use it as the default value in [google_api_lib/tasks.py](https://github.com/cardinalitypuzzles/cardboard/pull/753/files#diff-5ea7ae2b73cb0ddaa1f9bf676e71ab5a19d755c7eea5a80e6da1df1885d7a8f0L112), probably should clean this up or make an issue to clean it up later.